docs/pages/management/security.mdx

```diff
@@ -8,4 +8,7 @@ layout: tocless-doc
   <li>
     [Reducing the blast radius of attacks](./security/reduce-blast-radius.mdx). Prevent attackers from accessing your infrastructure even if they manage to obtain passwords or certificates.
   </li>
+  <li>
```

Comment:
- It's great that there's another guide in the "Security" section. The first guide in that section was my first assignment at Teleport!

- While we're here, maybe we could change the HTML to Markdown for consistency with the rest of the docs.

Reply:
I had that thought and didn't know if there was any particular reason why it's HTML. Will do!